### PR TITLE
Add nitro recipe

### PR DESCRIPTION
**Pull request by chambbj**

### CODE REVIEW

File: recipes/nitro/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "nitro" %}
+{% set version = "2.7.dev2" %}
+{% set sha256 = "cb00e5bd5d045f1bc333e1e054272c206c9910d008698b9cd7f2d67153a32ee2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: 2.7dev-2.tar.gz
```
Comment (mariusvniekerk) [inline recipes/nitro/meta.yaml]: you might want to use the `"{{ version }}` jinja tag here and in the url to make updating this recipe easier in the future

**Comment by chambbj** [comment]
@mariusvniekerk Jinja version tag updated.

File: recipes/nitro/bld.bat [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+cmake -G "NMake Makefiles" ^
+      -DCMAKE_INSTALL_PREFIX:PATH="%LIBRARY_PREFIX%" ^
+      -DCMAKE_BUILD_TYPE:STRING=Release ^
+      %SRC_DIR%
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
```
Comment (ocefpaf) [inline recipes/nitro/bld.bat]: Are there any tests, `ctest`, that we can run here?

File: recipes/nitro/build.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+set -x
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=$PREFIX ..
```
Comment (ocefpaf) [inline recipes/nitro/build.sh]: You can use the `$SRC_DIR` to be consistent with `bld.bat`.
Also, same question about the tests.

**Comment by chambbj** [comment]
@ocefpaf No tests, but I have fixed ${SRC_DIR} here as well.

**Pull request merged**